packages/commonwealth/client/scripts/views/components/component_kit/cw_sidebar_menu.tsx

```diff
@@ -121,26 +124,19 @@ export const CWSidebarMenuItem = (props: CWSidebarMenuItemProps) => {
       >
         {/*// @ts-expect-error <StrictNullChecks/>*/}
         <CommunityLabel community={item} />
-        {app.isLoggedIn() && roles.length > 0 && (
+        {app.isLoggedIn() && account && (
           <div className="roles-and-star">
             <User
               avatarSize={18}
               shouldShowAvatarOnly
-              userAddress={roles?.[0]?.address}
-              userCommunityId={
-                roles?.[0]?.address_chain || roles?.[0]?.community_id
-              }
-              shouldShowAsDeleted={
```

Comment:
I think this should not be deleted?

Reply:
It's irrelevant here -- this is a shouldShowAvatarOnly User (i.e. the little icons in the sidebar menu) and shouldShowAsDeleted only affects the text output ("Anonymous" vs "Deleted").